kubetest/extract.go

```diff
@@ -45,6 +45,7 @@ const (
 	version             // v1.5.0, v1.5.0-beta.2
 	gcs                 // gs://bucket/prefix/v1.6.0-alpha.0
 	load                // Load a --save cluster
+	bazel               // A pre/postsubmit bazel build version, prefixed with bazel/
```

Comment:
I've said this elsewhere, but to close the loop on this comment, the fact that we store CI artifacts and bazel artifacts separately (in `gs://kubernetes-release-dev/ci` and `gs://kubernetes-release-dev/bazel`) means that we need some differentiator somewhere to know where to fetch from, since the version string looks like a normal CI version string.

How we handle that UX, however, I am completely open to suggestions for. Since we already had the `ci/` prefix for that extract strategy ([here](https://github.com/kubernetes/test-infra/blob/a97d5b8c0c166bced91f911f66eed523bf55976a/kubetest/extract.go#L74)), it seemed natural to me to add a new `bazel/` prefix, but I'd love to hear other options.